Backend/api/main.py

```diff
@@ -27,8 +23,20 @@ def index():
     # /rest endpoints
     BuildingRestView.register(
         app, route_base="/building", trailing_slash=False)
+    ClassRoomRestView.register(
```

Comment:
Ar fi frumos si astea sa fie dupa ordinea logica in loc de alfabetic ca in __init__

Reply:
rezolvat